_data/tool_and_resource_list.yml

```diff
@@ -64,7 +64,7 @@
   name: LeafNet
   registry:
     biotools: leafnet
-  url: https://leafnet.whu.edu.cn/
+  url: https://bio.tools/leafnet
```

Comment:
```suggestion
  url: https://github.com/zhouyulab/leafnet
```

Reply:
I would link to the github page (taken from the publication

Reply:
we are linking to it through 

```
  registry:
    biotools: leafnet
 ```
 
 That is why we never link to a registry through the url attribute

Reply:
Ok. I see now. I got your point. And agree now. Feel free to link to github.